keps/349-all-or-nothing/README.md

```diff
@@ -215,6 +221,16 @@ type WaitForPodsReady struct {
 	// enforced only if waitForPodsReady.enable=true. If unspecified, it defaults to 5min.
 	// +optional
 	TimeoutSeconds *int64 `json:"timeoutSeconds,omitempty"`
+
+	// replacementTimeoutSeconds defines optional time duration in seconds, relative to
```

Comment:
We need to make sure that when the timeout is not specified, the Job does not get evicted immediately (see comment https://github.com/kubernetes-sigs/kueue/pull/2725#discussion_r1745970954) .
Please add a note about that.

Reply:
Updated description of the field so the value defaults to 3mins